app/src/main/java/app/revanced/integrations/sponsorblock/SponsorBlockUtils.java

```diff
@@ -585,16 +583,15 @@ public static String exportSettings(Context context) {
                     categorySelectionsArray.put(behaviorObject);
                 }
             }
+            json.put("userID", SettingsEnum.SB_UUID.getString());
+            json.put("isVip", SettingsEnum.SB_IS_VIP.getBoolean());
+            json.put("serverAddress", SettingsEnum.SB_API_URL.getString());
             json.put("dontShowNotice", !SettingsEnum.SB_SHOW_TOAST_WHEN_SKIP.getBoolean());
-            json.put("barTypes", barTypesObject);
             json.put("showTimeWithSkips", SettingsEnum.SB_SHOW_TIME_WITHOUT_SEGMENTS.getBoolean());
             json.put("minDuration", SettingsEnum.SB_MIN_DURATION.getFloat());
             json.put("trackViewCount", SettingsEnum.SB_COUNT_SKIPS.getBoolean());
             json.put("categorySelections", categorySelectionsArray);
-            json.put("userID", SettingsEnum.SB_UUID.getString());
-            json.put("isVip", SettingsEnum.SB_IS_VIP.getBoolean());
-            json.put("lastIsVipUpdate", SettingsEnum.SB_LAST_VIP_CHECK.getLong());
```

Comment:
This setting is missing now. What is the motivation?

Reply:
Only SB_LAST_VIP_CHECK was removed (it's not a setting to import/export).

The other fields were rearranged, so the json export better matches the appearance of the browser extension.

Reply:
Is the setting `SB_LAST_VIP_CHECK` relevant elsewhere in the code?

Reply:
it's only a date of the last time vip status was checked.  It's only used to prevent checking for vip status all the time, and it's not relevant for importing or exporting (the browser extension does import or export this field)